packages/create-next-app/create-app.ts

```diff
@@ -190,7 +192,9 @@ export async function createApp({
     console.log('Installing packages. This might take a couple of minutes.')
     console.log()
 
-    await install(root, null, { useYarn, isOnline })
+    if (!skipInstall) console.log('Skipping dependency installation step.')
```

Comment:
This is a bit confusing, shouldn't it be like:
- if `skipInstall` is `true` then log that installation step will be skipped
- if `skipInstall` is `false` then install dependencies

```suggestion
    if (skipInstall) console.log('Skipping dependency installation step.')
```

Reply:
yes, you're right